compiler/Options.hs

```diff
@@ -58,6 +58,7 @@ data Options
         { files :: [FilePath]
         , import_dir :: [FilePath]
         , output_dir :: FilePath
+        , using :: [String]
```

Comment:
this will be misleading for the end user since this feature will not be supported right?